api/src/main/java/marquez/common/Utils.java

```diff
@@ -172,4 +186,133 @@ public static Version newJobVersionFor(
             .getBytes(UTF_8);
     return Version.of(UUID.nameUUIDFromBytes(bytes));
   }
+
+  public static Version newDatasetVersionFor(
```

Comment:
Javadoc here and on other public methods

Reply:
+1